hack/test-cmd.sh

```diff
@@ -72,7 +72,8 @@ export ETCD_HOST=${ETCD_HOST:-127.0.0.1}
 export ETCD_PORT=${ETCD_PORT:-24001}
 export ETCD_PEER_PORT=${ETCD_PEER_PORT:-27001}
 
-os::util::environment::setup_all_server_vars "test-cmd/"
+os::cleanup::tmpdir
```

Comment:
Because tmpdir cleanup requires sudo.  Nothing about this flow involves sudo.

Tmpdir cleanup is really specific to one envorment, I don't even want us to be doing that.